extensions/interactions/ItemSelectionInput/validator.js

```diff
@@ -89,16 +93,58 @@ oppia.filter('oppiaInteractiveItemSelectionInputValidator', [
         baseInteractionValidationService.getAnswerGroupWarnings(
           answerGroups, stateName));
 
-      if (!defaultOutcome ||
-          $filter('isOutcomeConfusing')(defaultOutcome, stateName)) {
-        warningsList.push({
-          type: WARNING_TYPES.ERROR,
-          message: (
-            'Please clarify the default outcome so it is less confusing to ' +
-            'the user.')
+      var selectedChoices = [];
+      if (maxAllowedCount === 1) {
+        seenChoices.forEach(function(seenChoice, choiceIndex) {
```

Comment:
Is it possible to make this more similar to the corresponding logic in the other file?